source/jormungandr/jormungandr/scenarios/helper_classes/complete_pt_journey.py

```diff
@@ -113,6 +113,23 @@ def wait_and_build_crowflies(
     return res
 
 
+def tag_LEZ(journey):
+    tmp_list = [
+        not section.low_emission_zone.on_path
+        for section in journey.sections
+        if section.HasField('low_emission_zone')
+    ]
+    # all([]) == True it's not what we want :(
+    if tmp_list and all(tmp_list):
+        journey.low_emission_zone.on_path = False
+    elif any(
+        section.low_emission_zone.on_path
+        for section in journey.sections
+        if section.HasField('low_emission_zone')
+    ):
+        journey.low_emission_zone.on_path = True
+
+
```

Comment:
```suggestion
     for section in journey.sections:
         if section.HasField('low_emission_zone'):
             if section.low_emission_zone.on_path:
                 journey.low_emission_zone.on_path = True
                 break
```

Reply:
it won't work if none of the fallbacks has LEZ...